Tests/MapboxMapsTests/Foundation/Style/StyleTests.swift

```diff
@@ -120,6 +120,13 @@ final class StyleTests: XCTestCase {
         })
     }
 
+    func testGetAllLayerIdentifiersDoesNotTriggerAssertFor3DPuckLayer() {
+        styleManager.getStyleLayersStub.defaultReturnValue = [StyleObjectInfo(id: Puck3D.layerID, type: "model")]
+
+        // test fails if there is an assertion triggered here
+        _ = style.allLayerIdentifiers
```

Comment:
Can we check that all identifiers just don't include the 3D puck's one? We might have tests running on the Release config someday.

Reply:
Yes, makes sense, I updated the test